ygen/protogen.go

```diff
@@ -635,13 +645,25 @@ func safeProtoIdentifierName(name string) string {
 	return replacer.Replace(name)
 }
 
-// fieldTag returns a protobuf tag value for the entry e. The tag value supplied is
-// between 1 and 2^29-1. The values 19,000-19,999 are excluded as these are explicitly
-// reserved for protobuf-internal use by https://developers.google.com/protocol-buffers/docs/proto3.
+// fieldTag returns a protobuf tag value for the entry e.
```

Comment:
Update this comment too (for protoTagForEntry).

Reply:
Done, thanks, missed this before.